go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/kubev2v/migration-planner
 
-go 1.22.2
+go 1.23
```

Comment:
I would bump this to the version of the /ubi9/go-toolset, 1.23.6.

Reply:
Done